python/pyspark/sql/connect/client.py

```diff
@@ -141,7 +141,7 @@ def _build_metrics(self, metrics: "pb2.Response.Metrics") -> typing.List[PlanMet
     def sql(self, sql_string: str) -> "DataFrame":
         return DataFrame.withPlan(SQL(sql_string), self)
 
-    def collect(self, plan: pb2.Plan) -> pandas.DataFrame:
+    def toPandas(self, plan: pb2.Plan) -> pandas.DataFrame:
```

Comment:
I think we should remove this ... ? since this doesn;t exists in `SparkSession`

Reply:
cc @grundprinzip FYI

Reply:
We should make these methods private instead and see how they're used.

Reply:
The tricky part is that while the original session does not have this method we need a way to pass the plan to the client.

Reply:
I make it `private` but still have `data_frame` to call it. We need a way to pass the plan to the session/client.

We can see in the future if this can be replaced.